src/Serializer/MyTaskMetadataDtoNormalizer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace AppBundle\Serializer;
+
+use AppBundle\Api\Dto\MyTaskMetadataDto;
+use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
+
+class MyTaskMetadataDtoNormalizer implements ContextAwareNormalizerInterface, NormalizerAwareInterface
+{
+    use NormalizerAwareTrait;
+
+    private const ALREADY_CALLED = 'MyTaskMetadataDtoNormalizer_ALREADY_CALLED';
```

Comment:
what is the reason for these ALREADY_CALLED?

Reply:
See the explanation here: https://symfonycasts.com/screencast/api-platform2-security/normalizer-aware#avoiding-recursion-with-a-context-flag